src/components/board/client.ts

```diff
@@ -1,12 +1,13 @@
 import { Struct } from 'google-protobuf/google/protobuf/struct_pb';
 
+import { Duration } from 'google-protobuf/google/protobuf/duration_pb';
```

Comment:
[minor] see comment related to naming

```suggestion
import { Duration as PBDuration } from 'google-protobuf/google/protobuf/duration_pb';
```